gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/proposal/acceptProposal.ts

```diff
@@ -1,47 +1,158 @@
 import { Result } from '@chainlink/gauntlet-core'
-import { logger, BN, prompt } from '@chainlink/gauntlet-core/dist/utils'
-import { SolanaCommand, TransactionResponse, RawTransaction } from '@chainlink/gauntlet-solana'
-import { AccountMeta, PublicKey, Keypair } from '@solana/web3.js'
-import { Token, ASSOCIATED_TOKEN_PROGRAM_ID, TOKEN_PROGRAM_ID } from '@solana/spl-token'
+import { createHash } from 'crypto'
+import { logger, prompt, BN } from '@chainlink/gauntlet-core/dist/utils'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { PublicKey } from '@solana/web3.js'
+import { TOKEN_PROGRAM_ID } from '@solana/spl-token'
 import { utils } from '@project-serum/anchor'
-import { ORACLES_MAX_LENGTH } from '../../../../lib/constants'
 import { CONTRACT_LIST, getContract } from '../../../../lib/contracts'
-import { getRDD } from '../../../../lib/rdd'
-import { makeTx } from '../../../../lib/utils'
+import ProposeOffchainConfig, { OffchainConfig } from '../proposeOffchainConfig'
+import { serializeOffchainConfig } from '../../../../lib/encoding'
+import RDD from '../../../../lib/rdd'
 
 type Input = {
-  digest: Buffer
+  version: number
+  f: number
+  tokenMint: string
+  oracles: {
+    transmitter: string
+    signer: string
+    payee: string
+  }[]
+  offchainConfig: OffchainConfig
+  randomSecret: string
+}
+
+type DigestInput = {
+  version: BN
+  f: BN
+  tokenMint: PublicKey
+  oracles: {
+    transmitter: PublicKey
+    signer: Buffer
+    payee: PublicKey
+  }[]
+  offchainConfig: Buffer
+}
+
+type Proposal = {
+  owner: PublicKey
+  state: number
+  f: number
+  tokenMint: PublicKey
+  oracles: {
+    xs: {
+      transmitter: PublicKey
+      signer: {
+        key: Buffer
+      }
+      payee: PublicKey
+    }[]
+    len: number
+  }
+  offchainConfig: {
+    version: number
+    xs: Buffer
+    len: number
+  }
 }
 export default class AcceptProposal extends SolanaCommand {
   static id = 'ocr2:accept_proposal'
   static category = CONTRACT_LIST.OCR_2
 
   static examples = [
-    'yarn gauntlet ocr2:accept_proposal --network=devnet --state=EPRYwrb1Dwi8VT5SutS4vYNdF8HqvE7QwvqeCCwHdVLC',
+    'yarn gauntlet ocr2:accept_proposal --network=devnet --state=<STATE_ADDRESS> --rdd=<PATH_TO_RDD> <PROPOSAL_ID>',
```

Comment:
To keep consistent we should make the state address the argument, and proposalID as a flag